src/sentry/runner/decorators.py

```diff
@@ -25,3 +34,22 @@ def inner(ctx, *args, **kwargs):
             configure()
         return ctx.invoke(f, *args, **kwargs)
     return update_wrapper(inner, f)
+
+
+def log_level_option(f):
+    "Give ability to configure global logging level. Must be used before configuration."
+    import click
+    from functools import update_wrapper
+
+    @click.pass_context
+    @click.option('--loglevel', '-l', default=None,
+        help='Global logging level. Use wisely.',
+        envvar='SENTRY_LOG_LEVEL',
+        type=CaseInsensitiveChoice(LOG_LEVELS))
+    def inner(ctx, *args, **kwargs):
+        level = kwargs.pop('loglevel', None)
```

Comment:
I think this should be redone as:

```python
def inner(ctx, loglevel, *args, **kwargs):
    if loglevel is None:
        loglevel = 'WARN'
    os.environ.setdefault('SENTRY_LOG_LEVEL', loglevel)
    return ctx.invoke(f, *args, **kwargs)
```

We want to always set the `SENTRY_LOG_LEVEL` for children processes. And this prevents clobbering a `SENTRY_LOG_LEVEL` being set from the outside.